example-apps/typescript/src/authentication.ts

```diff
@@ -0,0 +1,42 @@
+import type { Authentication } from 'zapier-platform-core';
+
+import { SCOPES } from './constants';
+
+export default {
+  type: 'oauth2',
+  test: { url: 'https://auth-json-server.zapier-staging.com/me' },
```

Comment:
Suggest using the `API_URL` constant here and in the oauth2Config below.

Reply:
Done 👍